proposals/lip-0058.md

```diff
@@ -487,7 +531,7 @@ The function returns `True` if and only if there is an entry in the BFT Paramete
 ##### Execution
 
 ```python
-existBFTParameters(h):
+def existBFTParameters(h):
```

Comment:
Does `h` represent an height here? what type is it? if it is uint we still need to convert it internally

Reply:
The input parameters is described in the section Parameters above. Changed the line below to
```
if exists entry in BFT Parameters store with key uint32be(h):
```